shared/js/otSdkDriver.js

```diff
@@ -754,6 +744,29 @@ loop.OTSdkDriver = (function() {
       }.bind(this));
     },
 
+    _handleMessage: function(ev) {
+      try {
+        var message = JSON.parse(ev.data);
+        /* Append the timestamp. This is the time that gets shown. */
+        message.receivedTimestamp = (new Date()).toISOString();
+
+        if (message.type) {
```

Comment:
I suppose in implementing @Standard8's suggestion of a separate 'cursor' channel, we wouldn't need to check if a type exists when demuxing a message as 'text' will be exclusively chat messages and 'cursor' would be of position or click, etc.